src/langchain_google_alloydb_pg/model_manager.py

```diff
@@ -33,6 +33,7 @@ class AlloyDBModel:
     model_qualified_name: str
     model_auth_type: Optional[str]
     model_auth_id: Optional[str]
+    header_gen_fn: Optional[str]
```

Comment:
Would be this `generate_headers_fn` instead?
In `__acreate_model`, we loop over the kwargs and add optional configs dynamically, and per [this sample](https://cloud.google.com/alloydb/docs/ai/register-model-endpoint#anthropic) for "Auth header", the create model call looks like this:

```
CALL
  google_ml.create_model(
    model_id => 'MODEL_ID',
    model_provider => 'anthropic',
    model_request_url => 'REQUEST_URL',
    generate_headers_fn => 'google_ml.anthropic_claude_header_gen_fn');  -- ## not header_gen_function?
```

Reply:
```suggestion
    generate_headers_fn: Optional[str]
```